.gitignore

```diff
@@ -3,6 +3,7 @@ build
 node_modules/*
 .env
 .log
+.vscode/*
```

Comment:
I added this because new vscode started spitting logfiles into the project. I don't think I messed with any settings — lmk if you know something